src/app/examples/floating-action-button-example/floating-action-button-example.component.tns.scss

```diff
@@ -0,0 +1,4 @@
+/* Add mobile styles for the component here.  */
```

Comment:
You already have...consider removing comment?

Reply:
completely agree.